src/app/frontend/commons/commons_module.js

```diff
@@ -0,0 +1,11 @@
+import errorDialogService from './errordialog_service';
+
```

Comment:
I found a common folder after rebase. So, I moved the code to this folder. Do you agree?